packages/components/src/Dialog/Dialog.stories.tsx

```diff
@@ -72,20 +67,23 @@ export const Form: Story = {
     return (
       <Dialog.Trigger {...args}>
         <Button variant="primary">Open</Button>
-        <Dialog>
+        <Dialog closeButton>
           {({ close }) => (
             <>
-              <Dialog.Headline>Please log into account</Dialog.Headline>
-              <Stack space={4}>
+              <Dialog.Title>Please log into account</Dialog.Title>
+              <Dialog.Actions>
+                <Button variant="ghost" onPress={close}>
+                  Cancel
+                </Button>
+                <Button variant="primary">Login</Button>
+              </Dialog.Actions>
+              <Dialog.Content>
                 <TextField label="Username" />
                 <TextField label="Password" type="password" />
-                <Inline space={4}>
-                  <Button variant="ghost" onPress={close}>
-                    Cancel
-                  </Button>
-                  <Button variant="primary">Login</Button>
-                </Inline>
-              </Stack>
+              </Dialog.Content>
```

Comment:
from this example I'm not sure how  dialog actions should be used.

Reply:
and how its different from other parts of the dialog

Reply:
There are two reasons behind this : 
1-  Layout , to make sure that dilaog actions take the same place or area 
2-  Semantic clarity : it defines the purpose of the section as containing dialog actions .
@sarahgm

Reply:
And where should this actions be always placed? maybe with a better story it would be easier for me to understand

Reply:
With docs I mean how to use the different dialog components, and that they exists. It will be a breaking change so it should be clear what it is and what needs to change.